packages/core/docs/composables/composition-api.md

```diff
@@ -0,0 +1,179 @@
+# Composition API
+
```

Comment:
**suggestion** shouldn't we have a tip or something at the beginning pointing users who know CAPI into the next chapter or its officiousness?

Reply:
@filrak fixed!